src/index.ts

```diff
@@ -45,6 +45,23 @@ export class NovelCovid {
 
 	}
 
+	/**
+	 * @description Returns all the available countries.
+	 * @returns {Array<string>}
+	 */
+	async countriesNames(): Promise<Array<string>> {
+		this.countries().then(data => {
```

Comment:
as the function is async maybe use 
```ts
	async countryNames(): Promise<Array<string>> {
		const countries = await this.countries() as Country[] | null;
		return countries!.map(country => country.country).sort();
	}
```

Reply:
It looks good. I didn't knew that I could do that with async, I didn't had the chance to work with it.

Reply:
Forgot to ask, but what does the "!" do on countries?

Reply:
`!` is not null operator basically what countries could be null due to function overloading of other parameters out there so `!` tells the typescript complier that i am not null

Reply:
During development time typescript says it that it could be null but we know that its not null so we use `countries!.map(fn)`
![mu6a3QYe](https://flame.is-inside.me/mu6a3QYe.png)

Reply:
Oh... I see. I'm a C# dev, so `!` is basically `?`. Thanks for the tip!!

Reply:
The operator `?` it only validates undefined, instead `!` validate null and undefined

Reply:
`!` means "I promise you it aint null", `?` means "If it's null, return null, otherwise, do this. Both `!` and `?` could be used there.